File: CodeWars/2016/CW2016_02.java
import java.util.*;
import java.io.*;

public class prob02 {
	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		while(true){
			int orig = in.nextInt();
			if(orig==0){
				return;
			}
			int ans = 10000/orig;
			// This is a comment
			System.out.println(orig+" gallons per week will last " + ans+ " weeks");
		}
	}
}

File: CodeWars/2016/CW2016_02.py
for line in open("CW2014_02.in", "r"):
    gal = int(line)
    if gal == 0:
        break
    # print("%d gallons per week will last %d weeks" % (gal, int(10000/gal)))
    ans = int(10000/gal)
    print(gal+" gallons per week will last " + ans + " weeks")
